Opti_prop_T.m

```diff
@@ -0,0 +1,221 @@
+% |              V_inf   = Asymptotic speed     [m/s]                          |
+% |              Ct       = Thrust coefficients                                |
+% |Note        :                                                               |
+% ==============================================================================================
+
+%% INPUT
+%{
+N_blade =   [ ]
+R_hub   =   [%]
+R_tip   =   [m] 
+n_rpm   =   [rpm]
+V_inf   =   [m/s]
+T =         [N]
+P =         [kW]
+h =         [m]
+%}
+
+function [r_adim,a,a_first,dCt_dradim,w_0,k,Ct_new,dCp_dradim] = Opti_prop_T(N_blade,R_hub,R_tip,n_rpm,V_inf,Ct,h)
+%% CONVERSIONS
+%{
+From rpm to rps and rad/s
+%}
+n_rads = convangvel(n_rpm,'rpm','rad/s');
+n_rps = n_rpm/60;
+
+%% ATMO
+%{
+The aerospace toolbox is required in order to calculate the ISA parameters.
+It can be downloaded from the Mathworks website.
+%}
+[Temp a p rho] = atmosisa(h);
+
+%% Radius and adimensional values
+
+r = linspace(R_hub*R_tip,R_tip,1000);
+D = 2*R_tip;
+chi = (n_rads.*r)/V_inf;
+r_adim = r./R_tip;
+
+%% Step 1 First attempt values
+%{
+The two first attempt values of the axial induction w_0 and w_1 have been 
+set. w_0 = a*V_inf where a is the axial induction a = w_0/V_inf while 
+w_1 = 2*w_0. Two different first attempt values are necessary in order to 
+initialize the false position method.
+%}
+
+T    = Ct*rho*(n_rps^2)*(D^4);
+syms w0;
+eqn1 = w0 > 0;
+eqn2 = T == 2*rho*pi*(R_tip^2)*(V_inf+w0)*w0;
+eqn  = [eqn1 eqn2];
+S    = solve(eqn,w0,'ReturnConditions',true);
+w_0  = double(S.w0);
+w_1  = 1.5*w_0;
+T_1  = 2*rho*pi*(R_tip^2)*(V_inf+w_1)*w_1;
+Ct_1 = T_1/(rho*(n_rps^2)*(D^4));
+Ct_0 = 0;
+Ct_new = 0;
+
+%% Errors
+tao = 1*10^-8;
+error_0 = Ct-Ct_0;
+error_1 = Ct-Ct_1;
+
+%% False position method initialization
+%{
+The false position method has been chosen in ordert to implement the
+iterative cycle. The first value of the axial induction speed, has been
+calculated according to the classical method formulation by using the first
+attempt axial induction speed values.
+%}
+w = (w_1*error_0-w_0*error_1)/(error_0-error_1);
+k = 0; %cycle counter
+
+
+while abs(Ct-Ct_new) > tao
+%% Step 1
+%{
+In this step a calculation of:
+- chi = (OMEGA*r)/V_inf (an alternative non-dimensional radius)
+- a   = axial induction as function of chi 
+- a'  = rotational induction as function of chi 
+has been provided and plotted.
+%}
+a       = w/V_inf*((chi.^2)./((1+w/V_inf)^2+(chi.^2)));
+a_first = w/V_inf*((1+w/V_inf)./((1+w/V_inf)^2+chi.^2));
+
+%{
+The Prandtl correction function for finite blade number has been
+implemented. Lambda is  = chi^-1. According to the momentum theory: ...
+w_j = 2*w;
+%}
+lambda = (n_rads*R_tip/V_inf)^-1;
+F      = (2/pi)*acos(exp((N_blade/(2*lambda))*(r_adim-1)));
+
+%{
+The non-dimensional aerodynamic optimal load Gamma, already scaled with
+the Prandtl correction function is implemented.
+%} 
+GAMMA = (4*pi.*F.*(r_adim*R_tip).^2.*a_first*n_rads)/N_blade';

```

Comment:
Once you have multiplied `a_first` for `F`, the `F` is not anymore needed here, because the effect of the Prandtl correction function ha already been considered.